[2024-01-30T20:36:14.614-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-3/run_id=scheduled__2024-01-30T23:15:00+00:00/task_id=create_emr_cluster permission to 509
[2024-01-30T20:36:14.730-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-3/run_id=scheduled__2024-01-30T23:15:00+00:00/task_id=create_emr_cluster permission to 509
[2024-01-30T20:36:14.785-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: how-desafio-3.create_emr_cluster scheduled__2024-01-30T23:15:00+00:00 [queued]>
[2024-01-30T20:36:14.831-0300] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: how-desafio-3.create_emr_cluster scheduled__2024-01-30T23:15:00+00:00 [queued]>
[2024-01-30T20:36:14.832-0300] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2024-01-30T20:36:14.903-0300] {taskinstance.py:1382} INFO - Executing <Task(EmrCreateJobFlowOperator): create_emr_cluster> on 2024-01-30 23:15:00+00:00
[2024-01-30T20:36:14.905-0300] {standard_task_runner.py:57} INFO - Started process 130 to run task
[2024-01-30T20:36:14.908-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'how-desafio-3', 'create_emr_cluster', 'scheduled__2024-01-30T23:15:00+00:00', '--job-id', '304', '--raw', '--subdir', 'DAGS_FOLDER/weather_dag.py', '--cfg-path', '/tmp/tmp_q7sfqfi']
[2024-01-30T20:36:14.909-0300] {standard_task_runner.py:85} INFO - Job 304: Subtask create_emr_cluster
[2024-01-30T20:36:14.996-0300] {logging_mixin.py:151} INFO - Changing /mnt/c/Users/filipe.macedo_ifood/Documents/how-desafio-03/airflow/logs/dag_id=how-desafio-3/run_id=scheduled__2024-01-30T23:15:00+00:00/task_id=create_emr_cluster permission to 509
[2024-01-30T20:36:14.999-0300] {task_command.py:416} INFO - Running <TaskInstance: how-desafio-3.create_emr_cluster scheduled__2024-01-30T23:15:00+00:00 [running]> on host 14111-NB.
[2024-01-30T20:36:15.146-0300] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Filipe' AIRFLOW_CTX_DAG_ID='how-desafio-3' AIRFLOW_CTX_TASK_ID='create_emr_cluster' AIRFLOW_CTX_EXECUTION_DATE='2024-01-30T23:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-30T23:15:00+00:00'
[2024-01-30T20:36:15.147-0300] {emr.py:742} INFO - Creating job flow using aws_conn_id: aws_default, emr_conn_id: emr_default
[2024-01-30T20:36:15.167-0300] {logging_mixin.py:151} WARNING - /usr/local/lib/python3.10/dist-packages/airflow/providers/amazon/aws/operators/emr.py:750 UserWarning: Unable to find Amazon Elastic MapReduce Connection ID 'emr_default', using an empty initial configuration. If you want to get rid of this warning message please provide a valid `emr_conn_id` or set it to None.
[2024-01-30T20:36:15.187-0300] {base.py:73} INFO - Using connection ID 'aws_default' for task execution.
[2024-01-30T20:36:15.188-0300] {connection_wrapper.py:340} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2024-01-30T20:36:16.803-0300] {emr.py:756} INFO - Job flow with id j-150DZ8BKNASBK created
[2024-01-30T20:36:17.335-0300] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=how-desafio-3, task_id=create_emr_cluster, execution_date=20240130T231500, start_date=20240130T233614, end_date=20240130T233617
[2024-01-30T20:36:17.456-0300] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-01-30T20:36:17.565-0300] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
